docs/user-guide/configure-apiml-zosmf-workflow.md

```diff
@@ -0,0 +1,113 @@
+# Configuring API ML with z/OSMF Workflows
+
+After you install Zowe, you can register and execute the z/OSMF workflows in the web interface to perform a range of
```

Comment:
It seems that we should state that user's should use these workflows to configure API ML as a stand-alone component. Since no other configuration method exists to configure just API ML, I think we need to make sure that it is clear that this is the ONLY configuration method.

Reply:
well, technically they can still use "normal" zowe configuration workflow, it is just way more complicated, if your goal is  API ML, so I would not say ONLY, but PREFERED